packages/l2b/package.json

```diff
@@ -26,7 +26,6 @@
     "@l2beat/config": "workspace:*",
     "@l2beat/discovery": "workspace:*",
     "@l2beat/discovery-types": "workspace:*",
-    "@l2beat/protocolbeat": "workspace:*",
```

Comment:
This is actually used but not imported. See discovery-ui/main.ts

Reply:
good catch